src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -105,25 +118,22 @@ public Controls(Subsystems s) {
         if (INTAKE_ENABLED) {
             bindIntakeControls();
         }
-
         if (SHOOTER_ENABLED && SHOOTER_VISION_ENABLED) {
             bindShooterControls();
         }
     }
 
     // TODO these yay
     public void bindClimbControls() {
-
+        // bruh
     }
 
     public void bindDriveControls() {
-        resetDriveGyroButton.whenPressed(() -> {
-            subsystems.drivebaseSubsystem.resetGyroAngle(Rotation2.ZERO);
-        });
+        resetDriveGyroButton.whenPressed(() -> subsystems.drivebaseSubsystem.resetGyroAngle(Rotation2.ZERO));
     }
 
     public void bindIndexControls() {
-        if (SHOOTER_ENABLED && SHOOTER_VISION_ENABLED && INDEX_ENABLED) {
+        if (SHOOTER_VISION_ENABLED && INDEX_ENABLED) {
```

Comment:
? What was the reason for this change?

(Granted, `IndexShootCommand` does not strictly require the shooter subsystem, but still odd to only check one flag and not the other).

Reply:
will be confusing why an index specific command wont work when INDEX is enabled